eZ/Publish/Core/FieldType/Tests/Integration/BinaryBase/BinaryBaseStorage/BinaryBaseStorageGatewayTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\FieldType\Tests\Integration\BinaryBase\BinaryBaseStorage;
+
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\Core\FieldType\BinaryBase\BinaryBaseStorage\Gateway as BinaryBaseStorageGateway;
+use eZ\Publish\Core\FieldType\BinaryFile\BinaryFileStorage\Gateway\DoctrineStorage;
+use eZ\Publish\Core\FieldType\Tests\Integration\BaseCoreFieldTypeIntegrationTest;
+use eZ\Publish\SPI\Persistence\Content\Field;
+use eZ\Publish\SPI\Persistence\Content\FieldValue;
+use eZ\Publish\SPI\Persistence\Content\VersionInfo;
+use eZ\Publish\SPI\Tests\Persistence\FixtureImporter;
+use eZ\Publish\SPI\Tests\Persistence\YamlFixture;
+
+/**
+ * BinaryBase Field Type external storage gateway tests.
+ */
+class BinaryBaseStorageGatewayTest extends BaseCoreFieldTypeIntegrationTest
```

Comment:
This test is not getting executed (at least standalone, but I think it gets skipped in general)
```
eZ\Publish\Core\FieldType\Tests\Integration\BinaryBase\BinaryBaseStorage\BinaryBaseStorageGatewayTest is an integration test and requires setupFactory env setting. Use phpunit-integration-legacy.xml configuration to run it

 ./eZ/Publish/Core/FieldType/Tests/Integration/BaseCoreFieldTypeIntegrationTest.php:25
 

Test skipped because of an error in hook method
```

Reply:
Isn't it skipped because of not hooking phpunit-integration-legacy.xml? It's running correctly on my end (actually both are):
![image](https://user-images.githubusercontent.com/22300504/109679936-4cd32800-7b7c-11eb-9869-f988eae99062.png)

Reply:
Hmm, looks like this is some workaround I tried to do and has some legacy mistakes. Requires separate refactoring, +1 for now.